Libraries/NativeModules/specs/NativeAnimationsDebugModule.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+
+'use strict';
+
+import type {TurboModule} from 'RCTExport';
+import * as TurboModuleRegistry from 'TurboModuleRegistry';
+import {Platform} from 'react-native';
+
+export interface Spec extends TurboModule {
+  +startRecordingFps: () => void;
+  +stopRecordingFps: (animationStopTimeMs: number) => void;
+}
+
+export default (Platform.OS === 'android'
+  ? TurboModuleRegistry.getEnforcing<Spec>('AnimationsDebugModule')
+  : null);
```

Comment:
Shouldn't it be `undefined` instead?

Reply:
Please use `TurboModuleRegistry.get<Spec>('AnimationsDebugModule')` instead and no need to check the Platform.